examples/run_field_dep_justMw.py

```diff
@@ -1,151 +1,192 @@
+    this_dB = dB_settings
     for k in range(10):
         time.sleep(0.5)
-        if p.get_power_setting()>= this_dB: break
-    if p.get_power_setting() < this_dB: raise ValueError("After 10 tries, this power has still not settled")
+        if p.get_power_setting() >= this_dB:
+            break
+    if p.get_power_setting() < this_dB:
+        raise ValueError("After 10 tries, this power has still not settled")
     meter_powers = np.zeros_like(dB_settings)
     with xepr() as x_server:
         first_B0 = x_server.set_field(field_axis[0])
         time.sleep(3.0)
-        carrierFreq_MHz = gamma_eff * first_B0
+        carrierFreq_MHz = config_dict["gamma_eff_MHz_G"] * first_B0
         sweep_data = run_spin_echo(
-                nScans = nScans,
-                indirect_idx = 0,
-                indirect_len = len(field_axis),
-                adcOffset = adcOffset,
-                carrierFreq_MHz = carrierFreq_MHz,
-                nPoints=nPoints,
-                nEchoes = nEchoes,
-                p90_us = p90_us,
-                repetition = repetition_us,
-                tau_us = tau_us,
-                SW_kHz = SW_kHz,
-                output_name = output_name,
-                indirect_fields = ('Field','carrierFreq'),
-                ret_data = None)
-        myfreqs_fields = sweep_data.getaxis('indirect')
-        myfreqs_fields[0]['Field'] = first_B0
-        myfreqs_fields[0]['carrierFreq'] = carrierFreq_MHz
+            nScans=config_dict["nScans"],
+            indirect_idx=0,
+            indirect_len=len(field_axis),
+            adcOffset=config_dict["adc_offset"],
+            carrierFreq_MHz=carrierFreq_MHz,
+            nPoints=nPoints,
+            nEchoes=config_dict["nEchoes"],
+            p90_us=config_dict["p90_us"],
+            repetition=config_dict["repetition_us"],
+            tau_us=config_dict["tau_us"],
+            SW_kHz=config_dict["SW_kHz"],
+            output_name=filename,
+            indirect_fields=("Field", "carrierFreq"),
+            ret_data=None,
+        )
+        myfreqs_fields = sweep_data.getaxis("indirect")
+        myfreqs_fields[0]["Field"] = first_B0
+        myfreqs_fields[0]["carrierFreq"] = config_dict["carrierFreq_MHz"]
         for B0_index, desired_B0 in enumerate(field_axis[1:]):
             true_B0 = x_server.set_field(desired_B0)
-            logging.info("My field in G is %f"%true_B0)
+            logging.info("My field in G is %f" % true_B0)
             time.sleep(3.0)
-            new_carrierFreq_MHz = gamma_eff*true_Bo
-            myfreqs_fields[B0_index+1]['Field'] = true_B0
-            myfreqs_fields[B0_index+1]['carrierFreq'] = new_carrierFreq_MHz
+            new_carrierFreq_MHz = config_dict["gamma_eff_MHz_G"] * true_B0
+            myfreqs_fields[B0_index + 1]["Field"] = true_B0
+            myfreqs_fields[B0_index + 1]["carrierFreq"] = new_carrierFreq_MHz
             logging.info("My frequency in MHz is", new_carrierFreq_MHz)
```

Comment:
calculate resonance frequency for next field in field sweep using gamma_eff, add carrier frequency to list in indirect